src/styles/components/student-assignmentlist.styl

```diff
@@ -15,5 +15,11 @@
       margin-bottom: 1em
 
     .listitem > div:nth-child(1)
```

Comment:
Derp, of course. Do you want me to remove this before merging (assuming you're happy with the rest)?